src/main/java/frc/team2412/robot/commands/intake/AllInCommand.java

```diff
@@ -34,6 +34,6 @@ public void end(boolean interrupted) {
 
 	@Override
 	public boolean isFinished() {
-		return intakeSubsystem.getFeederSensor();
+		return !intakeSubsystem.getSensorOverride() || intakeSubsystem.getFeederSensor();
```

Comment:
I think this is different from the older version of the logic, if sensor override is true but feeder sensor is false, the old version would return true but the new version returns false.

Reply:
yeah you're right